client/my-sites/sharing/connections/services/google-my-business.js

```diff
@@ -63,19 +63,19 @@ export class GoogleMyBusiness extends SharingService {
 	};
 
 	componentWillMount() {
-		this.requestSettings( this.props );
+		this.requestKeyrings( this.props );
```

Comment:
Why can't we use `<QuerySiteKeyrings />` in this component?

Reply:
Because this component does not have a render method